src/docs/codelabs/implicit-animations.md

```diff
@@ -436,12 +441,16 @@ here are some suggestions for where to go next:
 [shape-shifting starter code]: /docs/codelabs/implicit-animations#shape-shifting-starter-code
 [shape-shifting complete]: /docs/codelabs/implicit-animations#shape-shifting-finished-code
 
+{% comment %} Flutter.dev {% endcomment %}
+
 [stateful widgets]: /docs/development/ui/interactive#stateful-and-stateless-widgets
 [staggered animations]: /docs/development/ui/animations/staggered-animations
 [hero animations]: /docs/development/ui/animations/hero-animations
 [animations tutorial]: /docs/development/ui/animations/tutorial
 [codelab]: /docs/codelabs
```

Comment:
We try to keep the links sorted alphabetically. I've been updating many pages with resorted links. :)

Reply:
No problem -- for your consideration I've submitted a proposal regarding long lists of links:
https://github.com/flutter/website/issues/3322